lib/game.rb

```diff
@@ -0,0 +1,101 @@
+class Game
+  attr_accessor :board, :player_1, :player_2
+  WIN_COMBINATIONS = [
+    [0, 1, 2], 
+    [3, 4, 5], 
+    [6, 7, 8], 
+    [0, 3, 6], 
+    [1, 4, 7], 
+    [2, 5, 8], 
+    [0, 4, 8], 
+    [2, 4, 6]
+    ]
+    
+    def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+      @player_1 = player_1
+      @player_2 = player_2
+      @board = board
+    end
+    
+    def current_player
+      if board.turn_count % 2 == 0
+        player_1
+      else
+        player_2
+      end
+    end
+    
+    def won?
+      WIN_COMBINATIONS.each do |winning_line|
+        if board.cells[winning_line[0]] == "X" && board.cells[winning_line[1]] == "X" && board.cells[winning_line[2]] == "X"
+          return [winning_line[0], winning_line[1], winning_line[2]]
+          elsif board.cells[winning_line[0]] == "O" && board.cells[winning_line[1]] == "O" && board.cells[winning_line[2]] == "O"
+          return [winning_line[0], winning_line[1], winning_line[2]]
+        end
+      end
+      false
```

Comment:
Here you can use [`Enumerable#detect`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-detect) to return the winning combination or `nil` if none is found.

```ruby
  def won?
    WIN_COMBINATIONS.detect do |combo|
      @board.cells[combo[0]] == @board.cells[combo[1]] &&
      @board.cells[combo[0]] == @board.cells[combo[2]] &&
      @board.taken?(combo[0] + 1)
    end
  end
```